src/components/data-table/QDataTable.vue

```diff
@@ -35,7 +35,7 @@
         <slot name="selection" :rows="selectedRows"></slot>
       </div>
     </div>
-
+    <slot name="customFilter"></slot>
```

Comment:
Are we in the situation where if slot exists then TableFilter should not be rendered? If so, wrap `<table-filter>` from below with this `<slot>`.